requirements.txt

```diff
@@ -6,3 +6,4 @@ rdflib==3.2.1
 argparse==1.1
 fastchardet==0.2.0
 mock==1.0.0
+simplejson==3.8.0
```

Comment:
Curious about this change. app-validator used to use simplejson instead of just json, and we dropped that when we switched to python 2.7 because the performance difference was negligible (and since simplejson was a compiled dependency, it was more annoying to keep it around)